backend/benefit/helsinkibenefit/urls.py

```diff
@@ -88,6 +88,7 @@
     path(
         "v1/handlerapplications/<str:application_id>/review/", ReviewStateView.as_view()
     ),
+    path("v1/print/<str:pk>/", application_views.PrintDetail.as_view()),
```

Comment:
If you put name="foo" here you can get the route by name later in tests for example like
reverse("foo", kwargs={"yourparam": "bar"}).